FAQ.md

```diff
@@ -0,0 +1,125 @@
+E                   - ${SCRIPT_DIR}/native-build.sh (Runs build.sh natively)
+E                   - ${SCRIPT_DIR}/configureBuild.sh
+E                     - ${SCRIPT_DIR}/sbin/common/constants.sh
+E                     - ${SCRIPT_DIR}/sbin/common/common.sh
+E                     - ${SCRIPT_DIR}/signalhandler.sh (rm container on SIGINT/SIGTERM)
+S                   - {buildOpenJDKViaDocker|buildOpenJDKInNativeEnvironment}
+```
+
+There is also some documentation in [CHANGELOG.md](CHANGELOG.md)
+
+## What are the prerequisites for a system used for builds?
+
+- The upstream OpenJDK build requirements are at https://wiki.openjdk.java.net/display/Build/Supported+Build+Platforms
+- The AdoptOpenJDK levels we build on are at https://github.com/AdoptOpenJDK/openjdk-build/wiki/%5BWIP%5D-Minimum-OS-levels
+  although anything with comparable equivalent or later C libraries should work ok (in particular we have built on most current Linux distros without issues)
+
+In terms of compilers, these are what we currently use for each release:
+
+| Version | OS      | Compiler |
+|---------|---------|----------|
+| JDK8    | Linux   | GCC 4.8 (HotSpot) GCC 7.6 (OpenJ9)                        |
+| JDK11+  | Linux   | GCC 7.5                            			|
+| JDK8    | Windows | VS2013 (12.0) (HotSpot) or VS2010 (10.0) (OpenJ9)		|
+| JDK11+  | Windows | VS2017							|
+| JDK8/11 | AIX     | xlC/C++ 13.1.3						|
+| JDK13+  | AIX     | xlC/C++ 16.1.0						|
+| JDK8    | macos   | GCC 4.2.1 (LLVM 2336.11.00				|
+| JDK11   | macos   | clang-700.1.81						|
+| JDK13+  | macos   | clang-900.0.39.2						|
+
+All machines at AdoptOpenJDK are set up using the ansible playbooks from the
+[infrastructure](https://github.com/adoptopenjdk/openjdk-infrastructure) repository.
+
+## Adding a new major release to be built
+
+1. Create the new release repository under github.com/adoptopenjdk (generally `openjdk-jdkxx`)
+2. Add the release to the list at [pipeline file](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build)
+3. Adjust the PR testing pipline [Example](https://github.com/AdoptOpenJDK/openjdk-build/pull/1394) to use the new release
+
+## Removing a major release once you've added a new one
+
+Unless the last release was an LTS one, you will generally want to remove one of the old versions after creating a new one. This can be done with `disableJob = true` in the release configuration files
+
+[Example](https://github.com/AdoptOpenJDK/openjdk-build/pull/1303/files)
+
+## How to enable/disable a particular build configuration
+
+1. Add/Remove it from the configuration files in https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/jobs/configurations
+2. if you're removing one and it's not just temporarily, you may want to delete the specific job from jenkins too
+
+[Example PR - removing aarch64 OpenJ9 builds](https://github.com/AdoptOpenJDK/openjdk-build/pull/1452)
+
+## How to add a new variant
```

Comment:
What is a variant?

Reply:
I called out "Dragonwell" as an example, and the `build-variant` parameter is documented elsewhere so I thought that would provide an adequate level of clarity, but I'll add some more words to the start of that section